ngraph/core/include/ngraph/op/interpolate.hpp

```diff
@@ -83,22 +85,27 @@ namespace ngraph
             private:
                 InterpolateAttrs m_attrs;
             };
-        }
+        } // namespace v0
 
-        namespace v3
+        namespace v4
         {
             class NGRAPH_API Interpolate : public Op
```

Comment:
We must add documentation for all classes and public methods in order to generate a good doxygen documentation.
Please add a documentation or create a ticket to document all new classes, structs, enums and methods after this PR.

Reply:
Done.